Makefile

```diff
@@ -82,7 +82,7 @@ bin/$(ARCH)/$(BIN): build-dirs
 	    -v $$(pwd)/.go:/go                                                 \
 	    -v $$(pwd):/go/src/$(PKG)                                          \
 	    -v $$(pwd)/bin/$(ARCH):/go/bin                                     \
-	    -v $$(pwd)/bin/$(ARCH):/go/bin/$$(go env GOOS)_$(ARCH)             \
+	    -v $$(pwd)/bin/$(ARCH):/go/bin/linux_$(ARCH)                       \
```

Comment:
Inside the Docker container, the operating system is _usually_ Linux. Are you thinking of the Microsoft Windows option here? It's hard to tell from reading press releases whether the container would be running within Windows, but running Linux inside, or whether the operating system _within_ the container is Windows too.